envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -0,0 +1,168 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwt_authn.v2;
+
+import "envoy/api/v2/route/route.proto";
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+// JSON Web Token (JWT) token format for authentication is defined
+// `here <https://tools.ietf.org/html/rfc7519>`_.
+// See `OAuth2.0 <https://tools.ietf.org/html/rfc6749>`_ and
+// `OIDC1.0 <http://openid.net/connect>`_ for the authentication flow.
+//
+// Example,::
+//
+//     issuer: https://example.com
+//     audiences:
+//     - bookstore_android.apps.googleusercontent.com
+//       bookstore_web.apps.googleusercontent.com
+//     jwks_uri: https://example.com/.well-known/jwks.json
+//
```

Comment:
nit: might be worth adding `not-implemented-hide` to indicate that this is not yet implemented in Envoy.
See, for example: https://github.com/envoyproxy/data-plane-api/blob/master/envoy/config/filter/accesslog/v2/accesslog.proto#L30

Reply:
Done